Packages/Sequence-Unity/Sequence/SequenceFrontend/Scripts/UI/Demos/MarketplaceTile.cs

```diff
@@ -11,27 +12,28 @@ namespace Sequence.Demo
     public class MarketplaceTile : MonoBehaviour
     {
         private Image _image;
-        private TextMeshProUGUI _priceText;
+        [SerializeField ]private TextMeshProUGUI _nameText, _priceText;
         private CollectibleOrder _collectibleOrder;
-
+        private Button _interactButton;
         private void Awake()
         {
             _image = GetComponent<Image>();
-            _priceText = GetComponentInChildren<TextMeshProUGUI>();
+            _interactButton = GetComponent<Button>();
+            _interactButton.onClick.AddListener(() => SequenceSampleUI.instance.OpenSeeMarketplaceDetailsPanelWithDelay(0, _collectibleOrder));
```

Comment:
Ok this one is 100% my fault. I think we should rename this UIPanel + UIPage to ViewMarketplaceListingsPanel

When I was reading this, I thought it was a typo of OpenSea as in OpenSeaMarketplaceListingsPanel as opposed to the intent which is Open(the)SeeMarketplaceDetailsPanel

What do you think?

Reply:
By the way, the change doesn't necessarily have to be in this PR, but if we're going to rename, we should make a note of it to do so before merging into master